internal/upload/vmware/vmware.go

```diff
@@ -17,9 +16,9 @@ type Credentials struct {
 	Datastore  string
 }
 
-// UploadImage is a function that uploads a stream optimized vmdk image to vSphere
+// ImportVmdk is a function that uploads a stream optimized vmdk image to vSphere
```

Comment:
I don't think `Import` is the right thing to call these. To me that implies importing into osbuild-composer, not uploading to an external service.

Reply:
So govc also has datastore.upload, which is why I renamed it to the govc command it calls, which is import.*. That said, happy to rename it to Upload(Ova|Vmdk). Mostly wanted to make the distinction between the two.

Reply:
Reasonable :) I'll change it after it's unblocked.

Reply:
FWIW, `Import` doesn't sound strange or wrong to me, but I'm fine either way.